### DISCUSSION TITLE
GitHub Page is not showing

### DISCUSSION THREAD

**Discussion by Kse-Ger** [community/community#28090] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi! I made my GitHub page using next.js and deployed it. But after changing it, it didn't update anymore. I pushed several changes, first about 2 weeks ago, nothing changed. 
Here is my repo: https://kse-ger.github.io/
Done a lot of research, cleared cache, tried in different browsers. Nothing helps

**Comment by airtower-luna** [comment]
I took a look at [the most recent build of your page](https://github.com/Kse-Ger/Kse-Ger.github.io/runs/7634569894?check_suite_focus=true), and at your repository. What seems to be going on is:

* Your repository doesn't contain an `index.html` file, nor any other HTML file (which means it can't work as a website).
* The default Jekyll processing for Pages turns the `README.md` into an `index.html` file.
* Result: You see an HTML rendering of your README file.

I'm not familiar with next.js so I can't say *how*, but you'll have to figure out how to build the HTML part of your website.

Also, because you're obviously not using Jekyll, you might want to [disable the Jekyll processing](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#static-site-generators). That won't solve other problems, but avoid possible conflicts later.

**Reply by airtower-luna** [reply]
As a side note, the author email address on your laptop doesn't seem to be set correctly, and your commits not recognized as connected to your account. See [Setting your commit email address](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address) for  fixing that.

**Comment by jfullstackdev** [comment, marked as answer]
seems this is fixed by now 

![image](https://user-images.githubusercontent.com/47092464/182660701-2ad6def2-5fb7-4a36-ac76-56740b245821.png)

![image](https://user-images.githubusercontent.com/47092464/182660789-4c40fd63-af1d-450b-a80a-bca75f78ca33.png)

enjoy further editing your page
